packages/@cdktf/cli-core/src/lib/init.ts

```diff
@@ -46,6 +46,16 @@ export type InitArgs = {
   silent?: boolean;
 };
 
+export const templatesDir = path.join(__dirname, "..", "..", "templates");
+
+const availableTemplates = fs
+  .readdirSync(templatesDir)
+  .filter((x) => !x.startsWith("."));
+export const templates: string[] = [];
+for (const template of availableTemplates) {
+  templates.push(template);
+}
```

Comment:
Is there a specific reason to writing it like this? instead of assigning `availableTemplates`?

Reply:
No idea, I just move the code. Don’t experiment in this pr 😅

Reply:
It was my code originally 😂😂😂 Case closed, I'm the idiot 🤝